code/logger/logFile.cpp

```diff
@@ -0,0 +1,24 @@
+#include "logFile.h"
+#include <stdexcept>
+
+
+LogFile::LogFile(const std::string& fileName) {
+	file.open(fileName, std::ofstream::out | std::ofstream::trunc);
+	
+	if (!file.is_open())
+		throw std::runtime_error(std::string("Could not open the log file:") + fileName);
```

Comment:
```suggestion
		throw std::runtime_error(std::string("Could not open the log file: ") + fileName);
```
Maybe add space for readability, but that's just a matter of taste.

Reply:
Done